### PR TITLE
Modified existing README.md

### PR DESCRIPTION
**Pull request by 96RadhikaJadhav**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,11 +1,16 @@
 # Versity
-Versity App will be a companion app for university students which will help them with academics and socials, the two aspects of university that our team believes an ‘all-rounder’ student should excel in.
+## 👨🏻‍💻 About the project
+
+- <p style="color: black;">Versity App will be a companion app for university students which will help them with academics and socials, the two aspects of university that our team believes an ‘all-rounder’ student should excel in.
```
Comment (kronaemmanuel) [inline README.md]: A bullet point and a `<p>` tag is unnecessary. Simple markdown is fine.

File: README.md [hunk 2]
```diff
@@ -1,11 +1,16 @@
 # Versity
-Versity App will be a companion app for university students which will help them with academics and socials, the two aspects of university that our team believes an ‘all-rounder’ student should excel in.
+## 👨🏻‍💻 About the project
```
Comment (kronaemmanuel) [inline README.md]: There need not be any special characters before 'About the project'. A simple heading is fine

File: README.md [hunk 3]
```diff
@@ -1,11 +1,16 @@
...
+
+- <p style="color: black;">Versity App will be a companion app for university students which will help them with academics and socials, the two aspects of university that our team believes an ‘all-rounder’ student should excel in.
 
 For academics, our app will help students organize their daily schedule. In our university, students are part of classes which have the same schedule, hence the users will be able to join their class’s group in the app. The class’s schedule will show up in the user’s personal calendar view.
 
 For socials, our app will help students be an active part of different societies in their university. It is usually hard to keep track of all the different events, meetings, practices of the societies/clubs that one has joined. Through our app, the society heads can make groups for the society and 
-Other than these two aspects, our users will also be able to add items to their personal schedules which will also appear in their calendar view.
+Other than these two aspects, our users will also be able to add items to their personal schedules which will also appear in their calendar view. :)</p>
+
+## 🤔 How to contribute
```
Comment (kronaemmanuel) [inline README.md]: Just 'How to contribute' is fine

File: README.md [hunk 4]
```diff
@@ -16,7 +21,59 @@ npm run dev
 # or
 yarn dev
 ```
-
 Open [http://localhost:3000](http://localhost:3000) with your browser to see the result.
 
 You can start editing the page by modifying `pages/index.js`. The page auto-updates as you edit the file.
+
+### : Star The Repo
```
Comment (kronaemmanuel) [inline README.md]: Probably should remove the ':' character. Just the heading is fine.

File: README.md [hunk 5]
```diff
@@ -16,7 +21,59 @@ npm run dev
...
+
+Star the repo by pressing the topmost-right button to start your wonderful journey.
+
+### : Fork The Repo
```
Comment (kronaemmanuel) [inline README.md]: Probably should remove the ':' character. Just the heading is fine.

File: README.md [hunk 6]
```diff
@@ -16,7 +21,59 @@ npm run dev
...
+
+You can get your own fork/copy of this repo by using the <b>Fork</b> button present at top-right of your screen.
+
+### : Clone it
```
Comment (kronaemmanuel) [inline README.md]: Probably should remove the ':' character. Just the heading is fine.

File: README.md [hunk 7]
```diff
@@ -16,7 +21,59 @@ npm run dev
...
+
+You need to clone (download) it to local machine using
+
+```sh
+$ git clone https://github.com/<Your_Github_Username>/versity.git
+```
+
+### : Create a new branch
```
Comment (kronaemmanuel) [inline README.md]: Probably should remove the ':' character. Just the heading is fine.

**Review by kronaemmanuel — CHANGES_REQUESTED** [review]
@96RadhikaJadhav, thank you so much for the PR. I have requested a few changes, kindly take a look at them and submit a commit to the same branch. After you have made the changes, please tag either @mrabdullahdev or me for a review.

**Comment by 96RadhikaJadhav** [comment]
> @96RadhikaJadhav, thank you so much for the PR. I have requested a few changes, kindly take a look at them and submit a commit to the same branch. After you have made the changes, please tag either @mrabdullahdev or me for a review.

@kronaemmanuel  Done the requested changes. Could you please review now and let me know if any changes required.

**Review by kronaemmanuel — APPROVED** [review]
Great work, @96RadhikaJadhav. Thank you for contributing.

**Comment by kronaemmanuel** [comment]
Closes #3

**Pull request merged**